harness/tests/integration_cases/test_raft.rs

```diff
@@ -5851,3 +5851,123 @@ fn test_switching_check_quorum() {
     }
     assert_eq!(sm.state, StateRole::Leader);
 }
+
+#[test]
+fn test_disable_proposal_forwarding() {
```

Comment:
Better follow the same test case as etcd. https://github.com/etcd-io/raft/blob/d6c02139d1de7eb4bea79a266ea94bab6a4e0864/node_test.go#L166-L192

Reply:
ok, updated

Reply:
Can you also move the test case to test_raw_node.rs?

Reply:
We align test cases to etcd. Test cases from raft_test.go are implemented in test_raft.rs, from node_test.go are implemented in test_raw_node.rs.

Reply:
ok, I moved the test case to raw node test.